src/components/entity.jsx

```diff
@@ -0,0 +1,126 @@
+import DocumentTitle from 'react-document-title'
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import config from '../config'
+import { entityShape } from '../openfisca-proptypes'
+
+
+class Entity extends React.Component {
+  static propTypes = {
+    countryPackageName: PropTypes.string.isRequired,
+    countryPackageVersion: PropTypes.string.isRequired,
+    intl: intlShape.isRequired,
+    entityId: PropTypes.string.isRequired,
+    entity: entityShape.isRequired,
+  };
+
+  
+  render() {
+    const { entityId, entity } = this.props
+
+    return (
+      <DocumentTitle title={entityId + ' — ' + this.props.intl.formatMessage({ id: 'appName' })}>
+        <section>
+          <h1><code>{ entityId }</code></h1>
+          {
+            entity.plural
+            ? <span>
+                <h5><code>{ entity.plural }</code></h5>
+              </span>
+            : <em><FormattedMessage id="noPlural"/></em>
+          }
+          { entity.description
+            ? <p className="description">{ entity.description }</p>
+            : <em><FormattedMessage id="noDescription"/></em>
+          }
+          { this.renderEntityMetadata(entity) }
+          {
+            entity.documentation &&
+            <div>
+              <span className="message"><FormattedMessage id="documentation"/></span>
```

Comment:
This should be a title, for semantics readability.

Reply:
I'm also wondering if we shouldn't put the documentation before the role descriptions. Right now, reading the page, we are not told what a menage is, but we have a table about roles that can be obscure at first sight.

Reply:
I would keep `Documentation` after table as the `description` is here to introduce the entity.
And at next visit, the user might be happy to avoid scrolling to get to roles list...